ckanext/geoimpact/templates/emails/invite_user.html

```diff
@@ -77,7 +77,10 @@ <h3>Account Details:</h3>
 
             <p>Wishing you a wonderful day!</p>
 
-            <p>geoimpact AG, Schulhausweg 7, 4571 Lueterkofen, Switzerland</p>
+            <div>
+                <div>SEP by <a href="https://www.geoimpact.ch/">geoimpact AG</a></div>
+                <div><a href="mailto:support@geoimpact.ch">support@geoimpact.ch</a> | +41 41 560 09 85</div>
+            </div>
 
             <div class="footer">
```

Comment:
p tag had 16px margins at top and bottom which created a space between footer and body. Adding a div between removes that space between footer and body. I would add margin-top to footer class

```
.footer {
    margin-top: 16px; /* <---- Add this line */
    text-align: center;
    color: #004016;
    padding: 10px;
    border-top: 1px solid #575756;
}
```

Reply:
without margin
![image](https://github.com/user-attachments/assets/ef9bfdc8-6105-404d-9375-520c0e672f84)

with margin
![image](https://github.com/user-attachments/assets/1097f0f0-9e12-499c-ac54-c53d7418a6ee)